keps/sig-api-machinery/20190927-insecure-backend-proxy.md

```diff
@@ -220,3 +221,60 @@ Similar to the `Drawbacks` section the `Alternatives` section is used to highlig
 Use this section if you need things from the project/SIG.
 Examples include a new subproject, repos requested, github details.
 Listing these here allows a SIG to get the process for these resources started right away.
+
+## Production Readiness
+
+  * Feature enablement and rollback
+    - How can this feature be enabled / disabled in a live cluster?
+      Once GA, this feature cannot be disabled at a cluster-wide level. It is the decision of the client whether or not
+      they will use the new API. 
+    - Can the feature be disabled once it has been enabled (i.e., can we roll
+      back the enablement)?
+      Not applicable.
+    - Will enabling / disabling the feature require downtime for the control
+      plane?
+      Not applicable.
+    - Will enabling / disabling the feature require downtime or reprovisioning
+      of a node?
+      Not applicable.
+    - What happens if a cluster with this feature enabled is rolled back? What
+      happens if it is subsequently upgraded again?
+      There is no data persisted as part of this API change, so there is no impact for upgrade or rollback.
+    - Are there tests for this?
+      There are integration tests that confirm the feature only takes effect when requested and works correctly.
+      You cannot add invalid kubelet certificates during e2e tests.
+  * Scalability
+    This is an additional value on an existing call and a change to TLS params. There isn't a meaningful impact on scalability.
```

Comment:
@johnbelamaric We should remove the exact questions from the KEP: https://github.com/kubernetes/enhancements/blob/master/keps/sig-architecture/20190731-production-readiness-review-process.md#proposal and replace with link - we don't want to maintain two versions.

Reply:
https://github.com/kubernetes/enhancements/pull/1408